doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -49,7 +49,7 @@ the assertion of a theorem using an assertion command like :cmd:`Theorem`. The
 list of assertion commands is given in :ref:`Assertions`. The command
 :cmd:`Goal` can also be used.
 
-.. cmd:: Goal @form
+.. cmd:: Goal @term
```

Comment:
This `@term` corresponds to what is after the colon in a `Lemma`, so could be `@type`.